src/controllers/user.js

```diff
@@ -88,34 +90,42 @@ export const updateById = async (req, res) => {
     }
 
     const updates = req.body
-    const allowedUpdates = [
-      'firstName',
-      'lastName',
-      'email',
-      'bio',
-      'githubUrl',
-      'mobile',
-      'specialism',
-      'imageUrl',
-      'cohortId'
-    ]
-
-    allowedUpdates.forEach((field) => {
-      if (updates[field] !== undefined) {
-        existingUser[field] = updates[field]
+    const updateUnchecked = async () => {
+      if (updates.password) {
```

Comment:
are you going to validate that the password matches the length, character requirements?

Reply:
what happens if I submit a password "a" via an api call?